src/ensembl/tests/fixtures/entity-viewer/gene.ts

```diff
@@ -26,7 +26,7 @@ import { TicksAndScale } from 'src/content/app/entity-viewer/gene-view/component
 
 export const createGene = (fragment: Partial<FullGene> = {}): FullGene => {
   const geneSlice = createSlice();
-  const transcript = createTranscript();
+  const transcripts = fragment.transcripts || [createTranscript()];
```

Comment:
Notice that your line 43 will be overwritten by line 57. What if you do `transcripts: [createTranscript()]` on line 43 and remove line 29?

Reply:
On line 29 we are avoiding a call to `createTranscript` when `fragment.transcripts` is present, in order to prevent `createGene` & `createTranscript` from calling each other recursively.

Reply:
Changed by updating the gene fixtures types and omiting gene from transcripts.